### DISCUSSION TITLE
Сharacter movement

### DISCUSSION THREAD

**Discussion by GammaSnaplight** [godot-engine-ru/shooter-3d#16] [Ideas]
Предлагаю идею по передвижению персонажа:
?- Отказаться от кнопки **бега** и **уворотов** в игре.
!- Я конечно не профи в программировании, особенно в GE, но мне кажется, что это принесет свои трудности, как минимум встревание в другие объекты, да и как-то данному геймплею не идет.
 Поэтому предлагаю дать персонажу только легкий бег, но чтобы компенсировать такую мобильность, персонаж может практически не замедляться при ношении какого-либо орудия и вовремя самой стрельбы. Легкое замедление (к примеру в 4%) даст должное ощущение, думаю объяснять не надо.
 Но в чем суть идеи, а в том, чтобы персонаж игры с каждым уровнем становился быстрее, а возможно и сильнее, как сам фактор адаптации к условию, также возможно это дает эффект "живого" персонажа. (Также можно переиграть какими-нибудь микстурами там или подобными вещами, которые будут давать данный эффект)
 Мое же предложение такое, с каждым уровнем давать 1-2% к скорости игрока и чтобы игрок хотя-бы догадался до этого, можно сделать, что персонаж в начале второго уровня скажет (и временами тоже) что чувствует прирост силы и т.д. И исходя от этого, заложить закладки (вьетнамские флешбеки) по уровням, которые дадут еще прироста к скорости/силе игрока, 
соответственно косвенно заставлять игрока осматривать локации и продлевать время игры. Как-то так.

29.01.2022 - Gamma

**Comment by slapin** [comment]
Таких трудностей как раз не возникнет. Разная скорость легко вписывается, если будут элементы тактики.

**Comment by Pigzinzspace** [comment]
у нас пока только 4 миссии, и мобы не меняются у нас прогресс и так достигается за счет получения более мощного оружия. нам надо думать не как его усилить а как создать ему больше проблем, чтоб усиление его оружия как то компенсировать.  если тебе не найти в какую ветку кидать предложения то кидай в эту https://github.com/godot-engine-ru/shooter-3d/discussions/12

**Reply by GammaSnaplight** [reply]
оке
